_posts/izayak/2016-01-21-Ruotong-Ch3-Conditional-Exercise.md

```diff
@@ -21,4 +21,7 @@ My questions:
 1. For exercise 3, for score less than 0 or greater than 10, I print 'Bad score' in the try part. Should that be put in except part? <\n>
 I also add a comment in the Trinket file to indicate that question's position.  
 
+Update:  
+This problem is solved when I do the Chapter4 Exercise.  
+https://github.com/silshack/spring2016/blob/gh-pages/_posts/izayak/2016-01-28-Ruotong-Ch4-Function-Exercise.md  
```

Comment:
Nice!  You'll want to link to the live version of that post, though: http://silshack.github.io/spring2016/Ruotong-Ch4-Function-Exercise.html